src/img/naked-mole-without-bg.svg

```diff
@@ -0,0 +1,19 @@
+<svg width="223" height="116" viewBox="0 0 223 116" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I am no expert about svgs but I know that we always try to keep viewbox to `0 0 30 30` (please, see other svg examples in the repo). Is there any reason we can't have viewbox with these values?

Reply:
I think the `0 0 30 30` here is not needed as this isn't an icon, so therefore it doesn't follow the 30x30 size standard that we have for icons in the dApp

Reply:
I thought we have it for all svgs, but the other naked mole doesn't have it indeed. Thank you for clarification.